### PR TITLE
[Catalog] Feature highlight should be triggered from a separate Button.

### PR DESCRIPTION
**Pull request by mohammadcazig**
Feature Highlight examples have a separate button to get triggered now.
Pivotal: https://www.pivotaltracker.com/story/show/157318200
before:
screen shot 2018-05-04 at 2 26 24 pm
screen shot 2018-05-04 at 2 26 21 pm

after:
screen shot 2018-05-14 at 11 48 01 am
screen shot 2018-05-04 at 2 26 21 pm

### CODE REVIEW

File: components/FeatureHighlight/examples/supplemental/FeatureHighlightExampleSupplemental.m [hunk 1]
```diff
@@ -45,14 +45,24 @@ - (void)viewDidLoad {
...
-  [self.button addTarget:self
-                  action:@selector(didTapButton:)
-        forControlEvents:UIControlEventTouchUpInside];
-  [MDCContainedButtonThemer applyScheme:buttonScheme toButton:button];
   [self.view addSubview:self.button];
 
+  MDCButton *actionButton = [[MDCButton alloc] init];
+  self.actionButton = actionButton;
+  [self.actionButton setBackgroundColor:[UIColor colorWithWhite:0.1f alpha:1]];
+  [self.actionButton setTitleColor:[UIColor whiteColor] forState:UIControlStateNormal];
```
Comment (jverkoey) [inline components/FeatureHighlight/examples/supplemental/FeatureHighlightExampleSupplemental.m]: Remove the color customizations.

File: components/FeatureHighlight/examples/supplemental/FeatureHighlightExampleSupplemental.m [hunk 2]
```diff
@@ -215,13 +224,23 @@ - (void)viewDidLoad {
...
-  [self.button addTarget:self
-                  action:@selector(didTapButton:)
-        forControlEvents:UIControlEventTouchUpInside];
-  [MDCContainedButtonThemer applyScheme:buttonScheme toButton:button];
   [self.view addSubview:self.button];
+
+  MDCButton *actionButton = [[MDCButton alloc] init];
+  self.actionButton = actionButton;
+  [self.actionButton setBackgroundColor:[UIColor colorWithWhite:0.1f alpha:1]];
+  [self.actionButton setTitleColor:[UIColor whiteColor] forState:UIControlStateNormal];
```
Comment (jverkoey) [inline components/FeatureHighlight/examples/supplemental/FeatureHighlightExampleSupplemental.m]: Remove here as well.

**Review by jverkoey — DISMISSED** [review]

**Comment by ianegordon** [comment]
Can you include a screenshot of the before view?

I know we previous had a square button that was overwritten with the FAB which looked a bit rough.  We should avoid this.  If we want to demo morphing, consider changing the icon, but not the button type / size.

**Review by jverkoey — DISMISSED** [review]
ian requested screenshot.

**Comment by mohammadcazig** [comment]
Updated the Screen Shots and fixed the morphing.

**Review by ianegordon — APPROVED** [review]

**Pull request merged**